dags/utils/slack_operator.py

```diff
@@ -15,74 +18,125 @@ def get_central_time_exec_data(context):
 
 
 def task_fail_slack_alert_critical(context):
-    slack_webhook_token = BaseHook.get_connection(SLACK_CONN_ID).password
     slack_msg = """
             <!channel> :red_circle: Critical Failure
             *Task*: {task}  
-            *Dag*: {dag} 
+            *DAG*: {dag} 
             *Execution Time*: {exec_date}  
             *Log Url*: {log_url} 
             """.format(
         task=context.get("task_instance").task_id,
         dag=context.get("task_instance").dag_id,
-        ti=context.get("task_instance"),
         exec_date=get_central_time_exec_data(context),
         log_url=context.get("task_instance").log_url,
     )
     failed_alert = SlackWebhookOperator(
         task_id="slack_critical_failure",
-        http_conn_id="slack",
-        webhook_token=slack_webhook_token,
```

Comment:
🙌 🙏 Nice to address this warning that we see in prod right now: `Provide webhook_token as hook argument deprecated by security reason and will be removed in a future releases. Please specify it in Slack Webhook connection.`